src/abstracts/SablierLockupBase.sol

```diff
@@ -462,12 +473,29 @@ abstract contract SablierLockupBase is
         _transfer({ from: currentRecipient, to: newRecipient, tokenId: streamId });
     }
 
+    /// @inheritdoc ISablierLockupBase
+    function withdrawFees() external override {
```

Comment:
I see that you have changed the design of this function:
1. The function is no long admin only. So anyone can call this function and send fees to `admin`
2. In case we rotate admin address, we also will have to move ETH fee from the old address to the new address.

If we make it admin only, we can specify a particular address and send eth fee to that regardless of admin change situation.

Just to confirm if this is intentional.

Reply:
Also in the [original specs](https://github.com/sablier-labs/v2-core/issues/1076), it is mentioned to allow admin to withdraw fee.